docs/topics/wasm/wasm-libraries.md

```diff
@@ -23,37 +13,18 @@ You can use one of the following repositories to add Kotlin libraries to your pr
   }
   ```
 
-* **Custom** Maven repository for experimental Kotlin/Wasm artifacts:
-
-  ```kotlin
-  // build.gradle.kts
-  repositories {
-      maven("https://maven.pkg.jetbrains.space/kotlin/p/wasm/experimental")
-  }
-  ```
-
-* **Custom** Maven repository for Compose Multiplatform dev artifacts:
-
-  ```kotlin
-  // build.gradle.kts
-  repositories {
-      maven("https://maven.pkg.jetbrains.space/public/p/compose/dev/")
-  }
-  ```
-
-| Library                       | Version          | Repository                                     |
-|-------------------------------|------------------|------------------------------------------------|
-| stdlib                        | %kotlinVersion%  | Maven Central                                  | 
-| kotlin-test                   | %kotlinVersion%  | Maven Central                                  |
-| kotlinx-coroutines            | 1.7.2-wasm3      | Custom for experimental Kotlin/Wasm artifacts  |
-| Compose Multiplatform         | 1.5.10-dev-wasm02| Custom for experimental Kotlin/Wasm artifacts  |
-| Compose Compiler              | 1.5.3            | Maven Central                                  |
-| kotlinx-serialization         | 1.6.1-wasm0      | Custom for experimental Kotlin/Wasm artifacts  |
-| Ktor                          | 3.0.0-wasm1      | Custom for experimental Kotlin/Wasm artifacts  |
-| kotlinx-atomicfu              | 0.22.0-wasm2     | Custom for experimental Kotlin/Wasm artifacts  |
-| kotlinx-collections-immutable | 0.4-wasm1        | Custom for experimental Kotlin/Wasm artifacts  |
-| kotlinx-datetime              | 0.4.1-wasm1      | Custom for experimental Kotlin/Wasm artifacts  |
-| skiko                         | 0.0.7.85-wasm01  | Custom for Compose Multiplatform dev artifacts |
+| Supported libraries           |
```

Comment:
This doesn't look so nice as a table. Maybe an unordered list would look better? Up to you though!

Reply:
This page is hidden and will be deleted in January, so I decided to leave it as is.